winui/Avatar-view/customization.md

```diff
@@ -1,10 +1,176 @@
 ---
 layout: post
 title: Customization in WinUI AvatarView control | Syncfusion
-description: Learn all about the customization features such as Height, Width, BorderBrush, Background, FontSize and CornerRadius in WinUI AvatarView control here.
+description: Learn all about the various customization features available in the WinUI AvatarView control, including border, background, gradient, font, and more.
 platform: WinUI
 control: AvatarView
 documentation: ug
 ---
 
-# Customization in WinUI AvatarView 
\ No newline at end of file
+# Customization in WinUI AvatarView 
+
+The `AvatarView` control offers a range of customization options, including the ability to modify its border, background, and more.
+
+## Border
+
+Customize the border of `AvatarView` control by using the `BorderBrush` and `BorderThickness` properties.
+
+**BorderBrush**
+
+You can customize the border color of the AvatarView control by using the `BorderBrush` property.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView  ContentType="AvatarCharacter" 
+                          AvatarCharacter="Avatar16"
+                          AvatarSize="ExtraLarge"
+                          BorderBrush="Red">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.ContentType = AvatarContentType.AvatarCharacter;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.AvatarCharacter = AvatarCharacter.Avatar16;
+avatarView.BorderBrush = new SolidColorBrush(Colors.Red);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom BorderBrush](avatarview_images/winui_avatarview_borderbrush.png)
+
+**BorderThickness**
+
+You can customize the border thickness of the AvatarView control by using the `BorderThickness` property.
```

Comment:
Customize the border thickness of the AvatarView control by using the `BorderThickness` property.